developers.properties

```diff
@@ -0,0 +1,6 @@
+# This file is here for 'hapi', because that "product" is missing the toplevel folder
+# (it contains the 'hapi' module directly)
+hedera-base@swirldslabs.com=Hedera Base Team
```

Comment:
This information [is already in this repo on `develop`](https://github.com/hashgraph/hedera-services/blob/263013048352c975a4c339ee255a59ccee93a282/gradle/plugins/src/main/kotlin/com.hedera.gradle.services-publish.gradle.kts#L33-L58).

This proposal only moves it into a separate file.

It is what ends up in the Metadata of the published Jars. For example here:
https://repo1.maven.org/maven2/com/hedera/hashgraph/hapi/0.53.5/hapi-0.53.5.pom

If we keep this information, I would like to move it into a separate file, so that it can be configured individually for each "product" (set of related modules).

However, we could consider to not have this and always just publish one generic "developer" entry in the metadata. We would then have less Mail addresses in the public and less information to maintain. That's something that can be discussed.

Reply:
Ah ok, makes sense. Thanks @jjohannes 💯

Reply:
Moved this aspect out to a separate PR to be integrated first. Please comment there is you have feedback: https://github.com/hashgraph/hedera-services/pull/15471